common/openapi.yml

```diff
@@ -453,15 +453,15 @@ components:
     JobResult:
       type: object
       description: |
-        The result of an async call that triggers a stairway job in TSPS.
+        The result of an async call that triggers a stairway job in Teaspoons.
```

Comment:
since this is user-facing text, we probably don't want to mention Teaspoons, and I don't think we really need to. how about we change this line to something like "The result of a call that triggers an asynchronous job"

Reply:
love it